tethysext/atcore/services/resource_spatial_manager.py

```diff
@@ -79,6 +86,7 @@ def create_extent_layer(self, datastore_name, resource_id, geometry_type=None):
 
         sql_context = {
             'resource_id': resource_id,
+            'srid': srid,
```

Comment:
There are two SRID's we are dealing with here. The SRID in the SQL is the SRID we'd like the geometry to be transformed to when it is queried. The SRID we specify in the create_layer call is the SRID of the given features. We don't want to lock these to be the same SRID. I'd say leave the SRID in the SQL hardcoded to transform to 4326 when we get the layer. But allow the user to specify the SRID of the features they have provided, defaulting to 4326. Make sense?

TL;DR: Don't pass the SRID argument to the SQL template.

Reply:
I was wrong on this. Sorry for the confusion.